content/variables/taxonomy.md

```diff
@@ -28,7 +28,7 @@ For example, the following fields would be available in `layouts/_defaults/terms
 : The singular name of the taxonomy (e.g., `tags => tag`)
 
 .Data.Plural
-: The plural name of the taxonomy (e.g., `tags => tags`)
```

Comment:
The above is the Markdown syntax for a definition list item, which we want to keep.

Reply:
@bep Thank you for your review, but the later commit fixed that.

Reply:
Ah, ok. Sorry.